JSON-connect/src/main.js

```diff
@@ -213,15 +324,37 @@ function getSchema(request) {
 }
 
 /**
- * Validates the row values. Only numbers and strings are allowed
+ *  Converts date strings to YYYYMMDDHH:mm:ss
+ *
+ * @param   {String} val  Date string
+ * @returns {String}      Converted date string
+ */
+function convertDate(val) {
+  var date = new Date(val);
+  return (
+    date.getUTCFullYear() +
+    ('0' + (date.getUTCMonth() + 1)).slice(-2) +
```

Comment:
Won't this always add a starting 0, even if the month is November or December? Same question for the month & day

Reply:
It slices the last two characters, so 012 becomes 12.

Reply:
Oh, duh. That's really handy!